pkgs/by-name/ge/gen/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage {
+  pname = "gen";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "NewDawn0";
+    repo = "gen";
+    rev = "v1.0.0";
+    sha256 = lib.fakeHash;
```

Comment:
Why is this using fake hashes?

Reply:
Oh my bad the fake hash was for testing, I'll fix it right away
I'm still somewhat new to nix, could you explain to me what the shellinit in nix is or link me the relevant resources on shellinit, so I can read up on it and apply the suggested changes.